Apsitvarkom.Models/Mapping/PollutedLocationProfile.cs

```diff
@@ -15,9 +15,9 @@ public PollutedLocationProfile()
         // PollutedLocationDTO to PollutedLocation
         CreateMap<PollutedLocationDTO, PollutedLocation>()
             .ForMember(dest => dest.Id, opt => opt
-                .MapFrom(src => Guid.Parse(src.Id)))
+                .MapFrom(src => Guid.Parse(src.Id ?? string.Empty)))
```

Comment:
As discussed, `src.Id` will be validated to not be null after we put explicit validation

Reply:
Possible null reference warning. Even though we will put it through validation, I'll keep it here for now.